src/core.js

```diff
@@ -56,14 +59,14 @@ define([
     datasources,
     util,
     jrUtil,
+    codemirror,
     debug
 ) {
     
     // Load these modules in the background after all runtime dependencies have
     // been resolved, since they're not needed initially.
     setTimeout(function () {
         require([
-            'codemirror',
```

Comment:
Was it not possible to load the `codemirror` dependencies here rather than above in the main code block?